provisioners/apt_proxy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "${apt_cache_host:-}" ]
+then
+    cat << EOF >> /etc/apt/apt.conf.d/00apt_proxy
```

Comment:
please rewrite this as `cat >/etc/apt/apt.conf.d/00apt_proxy <<'EOF'`.

Reply:
Ok. But I keep using `EOF` instead of `'EOF'` to keep variable substitution working.